src/components/Table/GridTable.test.tsx

```diff
@@ -2538,6 +2538,59 @@ describe("GridTable", () => {
       // Then the visible column session storage is defined using the `visibleColumnsStorageKey` prop
       expect(sessionStorage.setItem).toHaveBeenLastCalledWith("testStorageKey", '["name"]');
     });
+
+    it("respects setting inferSelectState to false", async () => {
+      // Given nested rows with two grand parents
+      const rows: GridDataRow<NestedRow>[] = [
+        simpleHeader,
+        {
+          // With one grand parent that sets `inferSelectedState: false`
+          ...{ kind: "parent", id: "p1", inferSelectedState: false, data: { name: "parent 1" } },
```

Comment:
@ipiedrahb - Once this is available, you'll want to just set `inferSelectedState: false` for the whole house template rows and you should be good to go!

Reply:
Hi @bdow 
Awesome, thank you for the support